var/spack/repos/builtin/packages/spdlog/package.py

```diff
@@ -28,4 +29,25 @@ class Spdlog(CMakePackage):
     version('0.10.0', '57b471ef97a23cc29c38b62e00e89a411a87ea7f')
     version('0.9.0', 'dda741ef8e12d57d91f778d85e95a27d84a82ac4')
 
-    depends_on('cmake@3.1:', type='build')
+    variant('shared', default=True,
+            description='Build shared libraries (v1.4.0+)')
```

Comment:
Eheh, not header only anymore (just read the description)?

Reply:
Can be both now :) Very nice for compile-time.